documentation/source/index.rst

```diff
@@ -1,37 +1,216 @@
-**********************************
-Welcome to Cocotb's documentation!
-**********************************
+##################################
+Welcome to cocotb's documentation!
+##################################
 
-Contents:
+..
+   This documentation tries to follow https://www.divio.com/blog/documentation/ (Daniele Procida)
+   Other media about the same topic:
+   - https://ep2018.europython.eu/media/conference/slides/get-your-documentation-right.pdf
+   - https://www.youtube.com/watch?v=t4vKPhjcMZg
+   - A good example: http://docs.django-cms.org/en/latest/contributing/documentation.html#contributing-documentation
+
+   See also https://github.com/cocotb/cocotb/wiki/Howto:-Writing-Documentation
+
+***************
+What is cocotb?
+***************
+
+**cocotb** is a *COroutine* based *COsimulation* *TestBench* environment for verifying VHDL/Verilog RTL using `Python <https://www.python.org>`_.
+
+cocotb is completely free, open source (under the `BSD License <https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_(%22BSD_License_2.0%22,_%22Revised_BSD_License%22,_%22New_BSD_License%22,_or_%22Modified_BSD_License%22)>`_) and hosted on `GitHub <https://github.com/cocotb/cocotb>`_.
+
+cocotb requires a simulator to simulate the HDL design
+and has been used with a variety of simulators on Linux, Windows and Mac OS.
+Please check the :ref:`simulator-support` page for specifics.
+
+A (possibly older) version of cocotb can be used live in a web-browser using `EDA Playground <https://www.edaplayground.com>`_.
+
+
+************************
+How is cocotb different?
+************************
+
+cocotb encourages the same philosophy of design re-use and randomized testing as UVM, however is implemented in Python rather than SystemVerilog.
```

Comment:
```suggestion
cocotb encourages the same philosophy of design re-use and randomized testing as `UVM <https://en.wikipedia.org/wiki/Universal_Verification_Methodology>`_, however is implemented in Python rather than SystemVerilog.
```